src/adagrams.js

```diff
@@ -1,8 +1,68 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const alphabet = 'AAAAAAAAABBCCDDDDEEEEEEEEEEEEFFGGGHHIJKLLLLMMNNNNNNOOOOOOOOPPQRRRRRRSSSSTTTTTTUUUUVVWWXYYZ';
+    const lettersInHand = [];
+
+    for (let i = 0; i < 10; i++) {
+      lettersInHand.push(alphabet.charAt(Math.floor(Math.random() * alphabet.length)));
+    }
+    return lettersInHand;
+  },
+
+  usesAvailableLetters(input, lettersInHand) {  
+    let result = true;
+    
+    for (let i = 0; i < (input.length); i++) {
+      if (lettersInHand.includes(input[i]) === false) {
+        result = false;
```

Comment:
since you used a traditional for loop you could simply `return false` here so you don't unnecessarily need to go through the whole loop.